ibm_mq/datadog_checks/ibm_mq/ibm_mq.py

```diff
@@ -44,6 +44,8 @@ def check(self, _):
             self.service_check(self.SERVICE_CHECK, AgentCheck.CRITICAL, self.config.tags)
             return
 
+        self._collect_metadata()
```

Comment:
```suggestion
        if self.is_metadata_collection_enabled():
            self._collect_metadata()
```

This is a convenience function that checks the base config in case the user has turned off the option.